notify/README.md

```diff
@@ -0,0 +1,101 @@
+# Collection - notify
+
+## API Endpoints
+
+|            Route             |                  Description                  |
+| :--------------------------: | :-------------------------------------------: |
+| [POST /notify](#post-notify) | send notification to your specified fcm token |
+|                              |
+
+## POST /notify
+
+    Send notification to your specified device.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **title** (required, string): notification title
+    - **body** (required, string): notification message
+    - **userId** (string): user who is assigning
+    - **groupRoleId** (string): group to whom they are assigning
```

Comment:
can a user send notification to a group only or it can be an individual as as well

Reply:
individual as well as group too